.github/workflows/push-go-runtime.yml

```
@@ -0,0 +1,72 @@
+# Workflows pertaining to the jsii/superchain Docker image
+name: Push Jsii Runtime Go
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  push-go-runtime:
+    name: Install Dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Set up Node 10
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: '10'
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install Dependencies
+        run: |-
+          # TypeScript project dependencies
+          yarn install --frozen-lockfile
+      - name: Align Versions
+        run: |-
+          ./scripts/align-version.sh
+      - name: Build dependencies
+        run: |-
+          yarn build \
+          --scope codemaker \
+          --scope @jsii/spec \
+          --scope @jsii/kernel \
+          --scope @jsii/runtime
+      - name: Build
+        run: |-
+          yarn build \
+          --scope @jsii/go-runtime
```

Would perhaps be more effective (and safer in the long run) to:

```suggestion
      - name: Build
        run: |-
          yarn build \
          --scope @jsii/go-runtime \
          --include-dependencies
```

---

Since `@jsii/go-runtime` devdepends on `jsii-pacmak` which devdepends on `@jsii/(dotnet|java)-runtime`, using `--include-dependencies` requires setting up those toolchains which I'd prefer to avoid. Removing `jsii-pacmak` from devDependencies also doesn't feel right, so I kept this for now.

As detailed in the [lerna docs](https://github.com/lerna/lerna/tree/main/core/filter-options#--include-dependencies), using `--include-dependencies` overrides `--scope` and `--ignore` flags.